source/features/delete-review-comments-faster.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function deleteComment(comment: HTMLElement): void {
+	select('.dropdown-menu .js-comment-delete > button', comment)!.click();
+}
+
+function onButtonClick(event: delegate.Event): void {
+	const comment = (event.target as HTMLElement).closest<HTMLElement>('.js-comment')!;
```

Comment:
When using `delegate`, always use `event.delegateTarget`, it's already the right type